pkg/mcpproxy/config.go

```diff
@@ -0,0 +1,165 @@
+package mcpproxy
+
+import (
+	"fmt"
+	"os"
+
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	TransportTypeHttp  = "http"
+	TransportTypeStdio = "stdio"
```

Comment:
Do we need `TransportStdio` ? If not, lets remove it

Reply:
We are using this for running evals against a `stdio` server. Our proxy will always run http though, currently